pkg/okteto/preview.go

```diff
@@ -81,7 +81,11 @@ type destroyPreviewMutation struct {
 	Response previewIDStruct `graphql:"destroyPreview(id: $id)"`
 }
 
-type listPreviewQuery struct {
+type listPreviewQueryWithLabels struct {
```

Comment:
I dont think we should include `WithLabels` in this struct. We could add more params to the query but we should add each param for the struct name